pkg/ext-proc/backend/provider.go

```diff
@@ -89,6 +90,15 @@ func (p *Provider) Init(refreshPodsInterval, refreshMetricsInterval time.Duratio
 		}
 	}()
 
+	// Periodically flush prometheus metrics for inference pool
+	flushPrometheusMetricsInterval := 10 * time.Second
```

Comment:
What do you think about promoting to a const at the top of the file for better visibility?

Reply:
Make it a flag and default is set to 5s.